en/components/excel_library.md

```diff
@@ -47,33 +45,43 @@ It's also possible to load a more minimal configuration of Excel Library whereby
 
 <div class="divider--half"></div>
 
-### Usage
-Now that the Excel Library module is imported, next step is to bind it to data.
-
-In order to create an `Workbook` object, you must first have data to bind it to. The following code snippet demonstrates how to create a simple data source.
-
-```typescript
-var data = ... TODO
-
-```
+### Load and Save Workbooks
+Now that the Excel Library module is imported, next step is to load a workbook.
 
-The following code demonstrates how to bind the `Workbook` object to the above data.
+In order to load and save `Workbook` object, you can utilize to load and save methods Excel. The following code is a class that exposes various methods such as load and save.
 
 ```typescript
 import { Workbook } from "igniteui-angular-excel/ES5/Workbook";
+import { WorkbookSaveOptions } from "igniteui-angular-excel/ES5/WorkbookSaveOptions";
 import { WorkbookFormat } from "igniteui-angular-excel/ES5/WorkbookFormat";
-import { Worksheet } from "igniteui-angular-excel/ES5/Worksheet";
-import { WorksheetTable } from "igniteui-angular-excel/ES5/WorksheetTable";
 
-TODO
+Workbook.load("ExcelWorkbook.xlsx", null, null);
+Workbook.save(workbook, "ExcelWorkbook.xlsx);
+
```

Comment:
are you sure we have these methods on workbook? If not then maybe we should just show:
var wb = new Workbook(format);
var ws2020 = wb.worksheets().add("Budget2020"); 
var ws2021 = wb.worksheets().add("Budget2021");